### PR TITLE
add kubelogin + updated helm and kubectl

### PR DESCRIPTION
**Pull request by pascal89**
Tested with updating the bitbucket pipeline with the new 1.2.0 version:
https://bitbucket.org/inforit-nl/workspace-base/pipelines/results/54

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,13 @@
 FROM mcr.microsoft.com/azure-cli
 
-# Based on dtzar/helm-kubectl
+# Install Kubectl + Kubectl login
```
Comment (Mastermindzh) [inline Dockerfile:3]: I would rephrase this to be clearer. It installs the cli tools which include kubectl etc but might include more (or less) now and in the future.

**Review by HBTeun — APPROVED** [review]
